contrib/build_rpm.sh

```diff
@@ -33,19 +33,19 @@ if [[ $pkg_manager == *dnf ]]; then
     PKGS+=(python3-devel \
         python3-varlink \
         )
-    # btrfs-progs-devel is not available in CentOS/RHEL-8
-    if ! grep -i -q 'Red Hat\|CentOS' /etc/redhat-release; then
-        PKGS+=(btrfs-progs-devel)
-    fi
-    # disable doc until go-md2man rpm becomes available
-    # disable debug to avoid error: Empty %files file ~/rpmbuild/BUILD/libpod-.../debugsourcefiles.list
-    export extra_arg="--without doc --without debug"
-else
-    if ! grep -i -q 'Red Hat\|CentOS' /etc/redhat-release; then
-        PKGS+=(golang-github-cpuguy83-go-md2man)
-    fi
 fi
 
+# btrfs-progs-devel is not available in CentOS/RHEL-8
+if ! (grep -i 'Red Hat\|CentOS' /etc/redhat-release | grep " 8" ); then
+    PKGS+=(golang-github-cpuguy83-go-md2man \
+        btrfs-progs-devel \
```

Comment:
was complaining it did not get those dep on centos 7 thus i moved the if here and i checked de version number